sstables_loader.cc

```diff
@@ -218,11 +240,31 @@ future<> sstable_streamer::stream(std::function<void(unsigned)> on_streamed) {
     co_await stream_sstables(full_partition_range, std::move(_sstables), std::move(on_streamed));
 }
 
+bool tablet_sstable_streamer::tablet_in_scope(locator::tablet_id tid) const {
+    if (_stream_scope == stream_scope::all) {
+        return true;
+    }
+
+    const auto& topo = _erm->get_topology();
+    for (const auto& r : _tablet_map.get_tablet_info(tid).replicas) {
+        if (_stream_scope == stream_scope::node && topo.is_me(r.host)) {
+            return true;
+        }
+        if (_stream_scope == stream_scope::rack && topo.get_rack(r.host) == topo.get_rack()) {
```

Comment:
`topo.get_location(r.host) == topo.get_location()`, since rack names might not be globally unique.

Reply:
Good point